CMSgov/onemac
# OY2 8959 - State User can view the updated submission pages

Story: https://qmacbis.atlassian.net/browse/OY2-8959
EndPoint: https://d2zbhgtcmuabdw.cloudfront.net
Figma: https://www.figma.com/file/OOtw6NirwsWqUlYkVW9pdz/Launch?node-id=1889%3A2333

### Changes
- Added /new, /newwaiver, and /newspa to the routing
- created ChoiceItem as the link with the title and description and arrow and styling
- created ChoiceList as the container for the ChoiceItems
- created newSubmission for the choice between SPA and Waiver
- created newSPA for the SPA choices
- created newWaiver for the Waiver choices
- created ChoiceList.test.js to test the building of the choices and the ability to route using the ChoiceItems

### Test
1. Log in as stateuseractive
2. If button is available, click button for New Submission, if not, change URL to be https://d2zbhgtcmuabdw.cloudfront.net/new
3. Look at page and compare to Figma
4. Click the choices and verify the paths
